schemas/2018-02-01/Microsoft.ServiceFabric.json

```diff
@@ -0,0 +1,983 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2018-02-01/Microsoft.ServiceFabric.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.ServiceFabric",
+  "description": "Microsoft ServiceFabric Resource Types",
+  "resourceDefinitions": {
+    "clusters": {
```

Comment:
>clusters [](start = 5, length = 8)

please update 2015-01-01/deploymentTemplate.json and 2014-04-01-preview/deploymentTemplate.json with the new resource types.

Reply:
@TianoMS Fixed.